src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.js

```diff
@@ -82,7 +82,7 @@ class ApprovalCommentForm extends React.Component<Props, State> {
     onFormValidSubmitHandler = (formData: any): void => {
         const { createComment, createTask, intl, updateTask, onSubmit, entityId } = this.props;
 
-        const commentText = this.getFormattedCommentText();
+        const { text: commentText, hasMention } = this.getFormattedCommentText();
```

Comment:
Does text have to be renamed?

Reply:
Nope, it was like that before, but it doesn't really add anything.